packages/build-electron/src/start/index.ts

```diff
@@ -59,14 +61,34 @@ export class ElectronStartBuilder implements Builder<ElectronStartBuilderSchema>
         const electron = require('electron');
 
         return new Observable((subscriber) => {
-            const child = proc.spawn(electron, [projectRoot, '--serve'], { stdio: ['pipe', 'inherit', 'inherit'] });
+            // console.log(this._serveAddress(builderConfig.options));
+            const options: proc.SpawnOptions = { stdio: ['pipe', 'inherit', 'inherit'] };
+            const args = [projectRoot, encodeURIComponent(this._serveAddress(builderConfig.options)), '--serve'];
```

Comment:
You can get rid of all this URIComponent stuff if you'll use environment variables just like what I did in this new commit:
https://github.com/ShPelles/electron-schematics/pull/20/files#diff-1f10877181d8ed47603d3ba5f57b8169R71

Reply:
You are right, I changed it.